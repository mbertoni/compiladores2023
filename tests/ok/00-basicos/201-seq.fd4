let (f:Nat -> Nat) = fun (n:Nat) ->
  let (x:Nat) = 20 in x+1 

let (y:Nat) = print "f 100 = " (f 100)