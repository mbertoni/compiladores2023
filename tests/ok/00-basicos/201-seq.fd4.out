f 100 = 21
